css/home.css

```diff
@@ -0,0 +1,455 @@
+/* Generated by http://css.spritegen.com CSS Sprite Generator */
```

Comment:
Please format it somehow. 
It should have the same indents. Use editorconfig or something other.